src/lib/LavalinkNode.ts

```diff
@@ -44,6 +44,10 @@ export class LavalinkNode {
      * The resume timeout
      */
     public resumeTimeout = 120;
+    /**
+     * Extra info attached to your node, not required and is not sent to lavalink, purely for you.
+     */
+    public state?: object;
```

Comment:
make this any. While i dont agree with any. state can be anything

Reply:
updated